doc/src/explanation/special_topics/finite_diff_derivatives.rst

```diff
@@ -49,7 +49,7 @@ If we let `x_{0} = x_{i}`, evaluate the series at `x_{i+1}=x_{i}+ h` and truncat
 
 .. math::
 
-	\big(\frac{dF}{dx}\big)_{x_{0}} \approx \frac{F_{i+1} - F_{i}}{h} + O(h)
+    \big(\frac{dF}{dx}\big)_{x_{0}} \approx \frac{F_{i+1} - F_{i}}{h} + O(h)
```

Comment:
I've always used indentation of 3 spaces in rst docs because it lines up with the first letter of the directive name. Should this be 3 spaces?

Reply:
You're right, it looks better with 3 spaces, I'm fixing it.